src/api/filesystem/async_io.rs

```diff
@@ -12,6 +12,13 @@ use std::time::Duration;
 
 use async_trait::async_trait;
 
+#[cfg(target_os = "linux")]
+use libc::{stat64};
+
+#[cfg(target_os = "macos")]
+use libc::{stat as stat64};
+
+
```

Comment:
we could reexport stat64 from transport/abi/linux_abi, so we could avoid conditional compilation here:)